packages/ember-application/lib/system/application.js

```diff
@@ -138,14 +146,16 @@ let librariesRegistered = false;
   append views inside it!
 
   To learn more about the events Ember components use, see
-  [components/handling-events](https://guides.emberjs.com/v2.6.0/components/handling-events/#toc_event-names).
+  [components/handling-events](https://guides.emberjs.com/v2.18.0/components/handling-events/#toc_event-names).
```

Comment:
Updated this as I was updating this file anyway. Let me know if it's an issue.

Reply:
cc @jenweber

Reply:
should be `https://guides.emberjs.com/current/components/handling-events/#toc_event-names`

Reply:
Added change.